docs/docs/activity.md

```diff
@@ -112,4 +115,4 @@ As you can see, a few features of the activity endpoint are missing.
 
 As a result, if you'd like to see a feature supported, feel free to create an issue and/or a pull request!
 
-[activity-scala]: https://github.com/47deg/github4s/blob/master/github4s/shared/src/main/scala/github4s/free/domain/Activity.scala
+[activity-scala]: https://github.com/47deg/github4s/blob/master/github4s/src/main/scala/github4s/domain/Activity.scala
```

Comment:
It seems this url launches a 404 error

Reply:
yep! The location of the resources are changing. There is no longer a "shared" directory.